### ISSUE TITLE
Syntax Highlighting failing on Annotations in python function defination

### ISSUE THREAD

**Issue by AmitGrowingData** [microsoft/vscode#84437]
- VSCode Version: 1.39.2
- OS Version: Windows 10

Steps to Reproduce:
This happens in python files. When we annotate functions with return type the code highlighting of the following function does not identify the function name.

```
def func1():
    return "ss"

def func2(): 
    return "func2"

def func3()->str: 
    return "func3"

def func4(): 
    return "func4"

def func5(): 
    return "func5"
```

![Capture](https://user-images.githubusercontent.com/54691623/68553276-dc2e6680-0473-11ea-8f6b-e2692ad77a8c.PNG)

Does this issue occur when all extensions are disabled?: Yes

How can we fix this?

**Comment by alexr00** [comment]
Hmm, I can't reproduce this:
![image](https://user-images.githubusercontent.com/38270282/68572902-db430800-0466-11ea-86d1-62b29cc2d449.png)
Are you sure you have all extensions disabled?

**Comment by AmitGrowingData** [comment]
Although I had disabled all extensions the issue seemed visible. I then opened a new folder and the problem disappeared. I guess it was a not cleared or something. After that, I enabled each one by one and found the issue to be in an extension "[Python for VS Code](https://github.com/tht13/Python-vscode)". This seems to be a 4 year last updated repository, which explains the issue. I am closing the issue here now, have put the details in case anyone faces a similar issue.
